scripts/stall-analyser.py

```diff
@@ -368,11 +375,13 @@ def main():
         # ?? ??:0
         if address_threshold:
             trace = list(dropwhile(lambda addr: int(addr, 0) >= address_threshold, trace))
-        tmin = args.minimum or 0
         if t >= tmin:
             graph.process_trace(trace, t)
 
     try:
+        if not graph:
+            print("No input data found. Please run `stall-analyser.py --help` for usage instruction")
+            exit(0)
```

Comment:
i'd suggest use `sys.exit()` instead. see https://docs.python.org/3/library/constants.html#constants-added-by-the-site-module . i am quoting the remarks in the document.

> They are useful for the interactive interpreter shell and should not be used in programs.

Reply:
done